.editorconfig

```diff
@@ -88,3 +88,7 @@ resharper_fsharp_multi_line_lambda_closing_newline = true
 fsharp_multi_line_lambda_closing_newline = true
 resharper_fsharp_strict_mode = false
 fsharp_strict_mode = false
+
+[data/generate.fsx]
```

Comment:
This keeps the formatting of the hardcoded table templates sane without too much modification to that file